projects/coordinator/src/main/java/org/batfish/coordinator/VersionAuthenticationFilter.java

```diff
@@ -0,0 +1,39 @@
+package org.batfish.coordinator;
+
+import com.google.common.base.Strings;
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import org.batfish.common.CoordConsts;
+import org.batfish.common.Version;
+
+/**
+ * This filter verify the access permissions for a user based on version provided in request header.
+ */
+public class VersionAuthenticationFilter implements ContainerRequestFilter {
```

Comment:
Why is this called an `Authentication` filter? It doesn't authenticate the client.